tests/org.eclipse.n4js.hlc.tests/pom.xml

```diff
@@ -105,17 +105,14 @@ Contributors:
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>${maven-surefire-plugin.version}</version>
-				<executions>
-					<execution>
-						<id>default-test</id>
-						<configuration>
-							<systemProperties>
-								<NODEJS_PATH>${NODEJS_PATH}</NODEJS_PATH>
-								<org.eclipse.n4js.defaultNodePath>${org.eclipse.n4js.defaultNodePath}</org.eclipse.n4js.defaultNodePath>
-							</systemProperties>
-						</configuration>
-					</execution>
-				</executions>
+				<configuration>
+					<argLine>-Xmx2048m</argLine>
```

Comment:
Can the general limit of the product be set to 2GB? It may be a good idea to increase it generally.